Makefile

```diff
@@ -14,7 +14,7 @@ DEV_ENV_IMAGE := quay.io/deis/go-dev:0.2.0
 DEV_ENV_WORK_DIR := /go/src/${REPO_PATH}
 DEV_ENV_CMD := docker run --rm -v ${PWD}:${DEV_ENV_WORK_DIR} -w ${DEV_ENV_WORK_DIR} ${DEV_ENV_IMAGE}
 DEV_ENV_CMD_INT := docker run -it --rm -v ${PWD}:${DEV_ENV_WORK_DIR} -w ${DEV_ENV_WORK_DIR} ${DEV_ENV_IMAGE}
-LDFLAGS := "-s -X main.version=${VERSION}"
+LDFLAGS := "-s -X main.version=${BUILD_TAG}"
```

Comment:
Actually I think `VERSION` is preferred here, at least if we allow the other components a vote. deis/workflow is the only one that uses `BUILD_TAG` instead, and it should probably be changed.